meta_capstone/ViewControllers/InGameViewController.m

```diff
@@ -42,11 +42,57 @@ - (void)viewDidLoad {
     if (self.game[@"tilesArray"] == nil) {
         //initialize dictionary
         self.wordCluePairs = [NSDictionary dictionaryWithObjectsAndKeys:
-                              @"this game, cross____", @"word",
-                              @"this game, _____word", @"cross",
-                              @"pink fish", @"salmon",
-                              @"not old", @"new",
+                              @"apple",@"apple",
+                              @"banana",@"banana",
+                              @"carrot",@"carrot",
+                              @"dinosaur",@"dinosaur",
+                              @"enigma",@"enigma",
+                              @"fishy",@"fishy",
+                              @"gorilla",@"gorilla",
+                              @"hippo",@"hippo",
+                              @"iguana",@"iguana",
+                              @"jaguar",@"jaguar",
+                              @"king",@"king",
+                              @"lame",@"lame",
+                              @"mirage",@"mirage",
+                              @"nicer",@"nicer",
+                              @"opal",@"opal",
```

Comment:
I thought the dictionary were words being pulled from the server. In addition, I thought the key/value pairs were word/clue pairs. If you implemented it that way later in PR dw about this, otherwise please explain

Reply:
i wanted to limit queries because they were getting really time intensive, so the wordlist is in the code. key/value pairs are supposed to be word/clue pairs, just updated that.